src/store.ts

```diff
@@ -1,34 +1,34 @@
 import { createStore, applyMiddleware } from 'redux'
 import thunk from 'redux-thunk'
-import { IConfig } from 'jolocom-lib'
 import { rootReducer, RootState } from './reducers'
 import { BackendMiddleware } from './backendMiddleware'
 import { IpfsLib } from 'src/lib/ipfs'
+import { entityList } from 'src/lib/storage/entities'
+import { ILibConfig } from 'jolocom-lib/js/types'
+import { defaultConfig } from 'jolocom-lib/js/defaultConfig'
 const { createReactNavigationReduxMiddleware } = require('react-navigation-redux-helpers')
 
-createReactNavigationReduxMiddleware('root', (state : RootState) => state.navigation)
-
-const jolocomLibConfig: IConfig = {
-  identity: {
-    providerUrl: 'https://rinkeby.infura.io/',
-    contractAddress: '0xd4351c3f383d79ba378ed1875275b1e7b960f120'
-  },
-  ipfs: {
-    host: 'ipfs.jolocom.com',
-    port: 443,
-    protocol: 'https'
-  },
+const jolocomLibConfig: ILibConfig = Object.assign({}, defaultConfig, {
   IpfsConnector: new IpfsLib()
-}
+})
 
+// TODO Break apart
 const config = {
   jolocomLibConfig,
-  fuelingEndpoint: 'https://faucet.jolocom.com/request'
+  fuelingEndpoint: 'https://faucet.jolocom.com/request',
+  typeOrmConfig: {
+    type: 'react-native',
+      database: 'LocalSmartWalletData',
+      location: 'default',
+      logging: ['error', 'query', 'schema'],
+      synchronize: true,
+      entities: entityList
+  }
```

Comment:
Added a comment on the library https://github.com/jolocom/jolocom-lib/issues/53#issuecomment-391990399